Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.12-slim
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
+
+RUN adduser \
+        --disabled-password \
+        --no-create-home \
+        django-user
+
+RUN chown -R django-user /files/media
+RUN chmod -R 755 /files/media

```

Comment:
These `RUN` commands can be combined to reduce the number of layers in the Docker image. Consider using `&&` to chain these commands together.